rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -358,6 +359,12 @@ void main_window::show_boot_error(game_boot_result status)
 	case game_boot_result::unsupported_disc_type:
 		message = tr("This disc type is not supported yet.");
 		break;
+	case game_boot_result::savestate_corrupted:
+		message = tr("Savestate data is corrupted or it's not an RPCS3 savestate.");
+		break;
+	case game_boot_result::savestate_version_unsupported:
+		message = tr("Savestate versioning data differes from your RPCS3 build.");
```

Comment:
```suggestion
		message = tr("Savestate version differs from your RPCS3 build.");
```

Reply:
It has multiple version entry, needs to be splled in plural.